nextflow.config

```diff
@@ -148,6 +150,25 @@ profiles {
         executor.cpus           = 4
         executor.memory         = 8.GB
     }
+    slurm {
+        process {
+            // Picked from the options above
+            executor   = 'slurm'
+
+            // The queue directory allows you to set the queue where jobs are scheduled when using a grid based executor
+            // queue      = 'standard'
+
+            // Number of retries when a job fails. Each subsequent run will use more memory in an attempt to solve memory errors.
+            maxRetries = 3
```

Comment:
Why hard code this in the profile if someone just wants to enable SLURM?

Reply:
good point